src/base/bittorrent/session.cpp

```diff
@@ -1486,6 +1486,23 @@ void Session::decreaseTorrentsPriority(const QStringList &hashes)
         torrentQueuePositionBottom(m_nativeSession->find_torrent(hash));
 }
 
+void Session::setTorrentsPriority(const QStringList &hashes, uint pos)
+{
+    // Prio torrents by hash order
+    foreach (const InfoHash &hash, hashes) {
+        TorrentHandle *const torrent = m_torrents.value(hash);
+        if (torrent && !torrent->isSeed()) {
+            try {
+                torrent->nativeHandle().queue_position_set(pos);
+                pos++;
```

Comment:
This time I will wait some time with my fix to prevent single line change pushs on daily base ;)